src/server/queries/queries.service.ts

```diff
@@ -0,0 +1,60 @@
+import { sql, eq } from "drizzle-orm";
+import { db } from "../db/db";
+import { shells } from "../db/schema";
+import { ContainerData } from "../../types/types";
+import Database from "better-sqlite3";
+
+export default class QueriesService {
+  public static async getShellIdsAsync(): Promise<number> {
+    const result = await db.select({ count: sql<number>`count(*)` }).from(shells);
+    return result[0].count;
+  }
+
+  public static async isPortAvailableAsync(port: number): Promise<boolean> {
```

Comment:
I’m thinking of refactoring this a bit so we have functions like `getShellFromPortAync` and `getShellFromNameAsync` instead of `isPortAvailableAsync` and `checkIfShellExistsAsync`.

Reply:
I see... Can you give some time to review the current changes? It is already a big pr lol